monero-outreach-docs/translations/fr/storyboard/wallet_quickstart_fr.md

```diff
@@ -0,0 +1,56 @@
+
+Certains portefeuilles fonctionnent en utilisant leur propre copie de la chaîne de blocs de Monero. On appelle également cela faire fonctionner un nœud complet. Cela vous permet de contrôler la rapidité de vos transactions. Cependant, vous pouvez vous appuyer sur un nœud distant sans risquer perdre vos Monero, vous dépendez alors de la disponibilité du nœud distant (vous serez toujours en mesure de recevoir des Monero, mais vous pourriez ne pas pouvoir en envoyer). Un autre atout du fonctionnement de votre propre nœud complet réside dans la fierté de contribuer au réseau Monero. Les nœuds complets sincères rendent Monero plus fort. Apprenez-en plus ici : [monero.how/how-to-run-monero-node](https://www.monero.how/how-to-run-monero-node).
+
+S'appuyer sur des nœuds distant, au contraire, permet (en utilisant se qui est communément appelé un portefeuille léger) de s'affranchir des ressources de stockage nécessaires à la chaîne de bloc Monero. La chaîne de bloc (non élaguée) fait plus de 50Go et continue de grossir. De nombreux portefeuilles s'appuyant sur des nœuds complets distants utilisent également moins de puissance de calcul et de bande passante réseau. Si ces ressources sont limités sur votre appareil, un portefeuille léger pourrait être préférable. Cela sera presque toujours le cas pour un portefeuille fonctionnant sur Smartphone.
+
+Il y a deux types de portefeuilles légers : à protection ou à partage de clef de lecture. Les portefeuilles légers à protection de clef gardent vos clefs de lecture confidentielles, et doivent donc télécharger la chaîne de bloc, ou du moins les portions plus récentes que la date de création du portefeuille, pour pouvoir déterminer le solde du portefeuille. Les portefeuilles légers à partage de clef, par opposition, partage vos clefs de lecture avec le serveur exécutant le nœud distant pour calculer le solde du portefeuille depuis celui-ci. Ces portefeuilles performants sont parfois appelés applications de portefeuilles. Le partage de vos clefs de lecture amoindrit la confidentialité si le serveur exécutant le nœud distant et connaissant les clefs de lecture n'est pas fiable car il permet l'observation des transactions entrantes. Remarquez, cependant, que des propriétaires de portefeuilles partagent parfois leurs clefs de lecture pour des besoins d'audit (elles ne permettent pas de transférer des Monero).
+
+Pour plus de sécurité (même en cas de compromission de votre appareil principal par un malware) des portefeuilles matériel spécialisés peuvent être utilisés. Les portefeuilles matériels sont de petits appareils électroniques interactifs qui stockent vos clefs privées et communique avec le logiciel sur l'appareil principal en n'utilisant que des informations publiques. Les portefeuilles matériels apporte une sécurité accrue pour un prix moyen entre 50€ et 150€.
```

Comment:
'Pour plus de sécurité' ---> 'Pour une sécurité maximale'
Reason: The original document states 'For maximum security' not 'For more security'

Reply:
I agree with the suggestion, but please do not try to stick that much to the English version. The translation has to respect what the author intended to write, be technically correct, but moreover needs to be "pleasant" to read in French. As long as both the first are respected, the last is the more important.

Reply:
The message is very different if you try to use 'Pour plus de sécurité' instead of 'Pour une sécurité maximale'. Reading again the english version, I think the author do think something like 'To achieve a maximum level of security'. Which is closer to my proposed correction.
Anyway, glad to see that we agree :)